impeller/renderer/backend/vulkan/context_vk.h

```diff
@@ -131,10 +133,16 @@ class ContextVK final : public Context,
   std::shared_ptr<FenceWaiterVK> GetFenceWaiter() const;
 
  private:
-  vk::UniqueInstance instance_;
+  struct UniqueDeviceHolder : public DeviceHolder {
```

Comment:
Avoid the "Unique" name for something that does not provide an interface similar to `std::unique_ptr`.  Call this something like `DeviceHolderImpl`.

Reply:
done